### PR TITLE
chore: LotteryBanner while preparing

### PR DESCRIPTION
**Pull request by Chef-Yogi**
demo

### CODE REVIEW

File: src/views/Home/components/Banners/LotteryBanner.tsx [hunk 1]
```diff
@@ -91,26 +92,40 @@ const LotteryCountDownTimer = () => {
...
     <S.Wrapper>
       <S.Inner>
         <S.LeftWrapper>
-          <StyledSubheading style={{ textShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)' }}>
-            <LotteryPrice />
-          </StyledSubheading>
-          <TimerWrapper>
-            <LotteryCountDownTimer />
-          </TimerWrapper>
+          {isLotteryOpening(status) ? (
```
Comment (memoyil) [inline src/views/Home/components/Banners/LotteryBanner.tsx:111]: Can we merge this two conditional blocks ?

File: src/views/Home/components/Banners/LotteryBanner.tsx [hunk 2]
```diff
@@ -67,7 +68,7 @@ const LotteryPrice: React.FC = () => {
   const prizeTotal = getBalanceNumber(prizeInBusd)
   const { t } = useTranslation()
 
-  if (status === LotteryStatus.OPEN) {
+  if (isLotteryOpening(status)) {
```
Comment (memoyil) [inline src/views/Home/components/Banners/LotteryBanner.tsx:71]: isLotteryLive maybe better name wdyt?

**Review by Chef-Cheems — COMMENTED** [review]
Wouldn't this show "Preparing" for a brief moment while we load lottery data?
We actually show pending lottery UI for a second on a lottery page too while lottery is data is not loaded. Maybe we can add new status to LotteryStatus enum like "UNINITIALIZED" 🤔

**Comment by Chef-Yogi** [comment]
@Chef-Cheems Got it will add a in initialize state and handle it

File: src/views/Home/components/Banners/LotteryBanner.tsx (hunk 1 above)
Comment (Chef-Yogi) [reply src/views/Home/components/Banners/LotteryBanner.tsx:111]: Done

File: src/views/Home/components/Banners/LotteryBanner.tsx (hunk 2 above)
Comment (Chef-Yogi) [reply src/views/Home/components/Banners/LotteryBanner.tsx:71]: Done

**Comment by Chef-Yogi** [comment]
@Chef-Cheems  I think I fix the “display the preparing while loading” issue.
I didn't add status for that, I handle this case by using the useLottery hook.
it provides the current round lottery isLoading status

**Review by Chef-Cheems — APPROVED** [review]

**Pull request merged**